src/GitHub.App/Services/GitClient.cs

```diff
@@ -405,19 +405,20 @@ public Task<bool> IsModified(IRepository repository, string path, byte[] content
             {
                 if (repository.RetrieveStatus(path) == FileStatus.Unaltered)
                 {
-                    var head = repository.Head[path];
-                    if (head.TargetType != TreeEntryTargetType.Blob)
```

Comment:
Given that you always want to return false if it's not a blob, and not having a head is equivalent to not being a blob, how about just doing `if (head?.TargetType != TreeEntryTargetType.Blob)` here and no other change?

Reply:
Updated to do it like this.